### PR TITLE
Fix 'Can't convert to color: type=0x2 error' on Session Feedback screen

### PR DESCRIPTION
**Pull request by hkurokawa**
## Issue
#287 

## Contents
- [x] 	Use `@color/theme500` instead of `?colorPrimary` for API<21

[Review on Reviewable](https://reviewable.io/reviews/konifar/droidkaigi2016/288)

### CODE REVIEW

File: app/src/main/res/drawable/clickable_primary.xml [hunk 1]
```diff
@@ -3,7 +3,7 @@
 
     <item android:state_pressed="false">
         <shape android:shape="rectangle">
-            <solid android:color="?colorPrimary" />
+            <solid android:color="@color/theme500" />
```
Comment (konifar) [inline app/src/main/res/drawable/clickable_primary.xml]: How about this?

```xml
<solid android:color="?attr/colorPrimary" />
```

Comment (konifar) [inline app/src/main/res/drawable/clickable_primary.xml]: This is used in `activity_contributor.xml`

```xml
        <android.support.v7.widget.Toolbar
            android:id="@+id/toolbar"
            android:layout_width="match_parent"
            android:layout_height="?attr/actionBarSize"
            android:background="?attr/colorPrimary"
            android:elevation="@dimen/elevation"
            app:elevation="@dimen/elevation"
            app:popupTheme="@style/ThemeOverlay.AppCompat.Light" />
```

Comment (hkurokawa) [inline app/src/main/res/drawable/clickable_primary.xml]: Oh, didn't notice that. Yes, it should be better. Thank you.

Comment (hkurokawa) [inline app/src/main/res/drawable/clickable_primary.xml]: :pensive: Unfortunately, the change seems not fixing the issue. The same error: `Can't convert to color: type=0x2 error`.

Comment (konifar) [inline app/src/main/res/drawable/clickable_primary.xml]: Hmm...OK! Your fix looks good!

**Comment by konifar** [comment]
LGTM!

**Comment by hotchemi** [comment]
LGTM!

**Pull request merged**